app/models/miq_cockpit_ws_worker.rb

```diff
@@ -0,0 +1,64 @@
+require 'miq_apache'
+
+class MiqCockpitWsWorker < MiqWorker
+  require_nested :Runner
+
+  APACHE_CONF_FILE = '/etc/httpd/conf.d/manageiq-cockpit.conf'.freeze
+  self.required_roles = ['cockpit_ws']
+  self.maximum_workers_count = 1
+
+  def friendly_name
+    @friendly_name ||= "Cockpit Worker"
+  end
+
+  def self.can_start_cockpit_ws?
+    @supports_cockpit_ws ||= MiqEnvironment::Command.is_linux? &&
+                             File.exist?('/usr/libexec/cockpit-ws') &&
+                             File.exist?('/usr/libexec/cockpit-ssh')
```

Comment:
Do these paths belong in the MiqCockpit module along with the hardcoded redirect URL and port?